omnigibson/simulator.py

```diff
@@ -803,13 +903,17 @@ def n_physics_timesteps_per_render(self):
             assert n_physics_timesteps_per_render.is_integer(), "render_timestep must be a multiple of physics_timestep"
             return int(n_physics_timesteps_per_render)
 
-        def step(self, render=True):
```

Comment:
what was the reason for removing the render flag?

Reply:
EDIT: nvm, I assume that the new `render_of_step` context manager is meant to handle this

Reply:
Yup